io.catenax.id_based_comment/1.0.0/gen/IdBasedComment.html

```diff
@@ -0,0 +1,1123 @@
+
+<!doctype html>
```

Comment:
You can remove completely the 'gen' folder as this will be generated at run time automatically.